releasenotes/notes/apm-rm-service-writer-8818e02f95020d2c.yaml

```diff
@@ -6,7 +6,7 @@
 #
 # Each section note must be formatted as reStructuredText.
 ---
-upgrade:
+enhancements:
```

Comment:
this is still listed as an `Upgrade` item in the changelog :)

Reply:
Whoops, fixed.